File: antlr3/org/softlang/company/Employee.java
package org.softlang.company;

/**
 * An employee has a name, an address, and a salary.
 */
public class Employee {

	private String name;
	private String address;
	private double salary;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

	public double getSalary() {
		return salary;
	}

	public void setSalary(double salary) {
		this.salary = salary;
	}
}

File: contributions/html5tree/server/classes/employee.php
<?php

    class Employee {
        
        public $id;
        public $name;
        public $address;
        public $salary;
        public $departments;
        
        public function getId() {
            return $id;
        }
        
        public function getName() {
            return $name;
        }
        
        public function getAddress() {
            return $address;
        }
        
        public function getSalary() {
            return $salary;
        }
        
        public function getDepartments() {
            return $departments;
        }
        
        public function setId($newId) {
            $this->id = $newId;
        }
        
        public function setName($name) {
            $this->name = $name;
        }
        
        public function setAddress($address) {
            $this->address = $address;
        }
        
        public function setSalary($salary) {
            $this->salary = $salary;
        }
        
        public function setDepartments($newDepartments) {
            $this->departments=$newDepartments;
        }
    }
?>
